.gitpod.Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM gitpod/workspace-base
+
+# Installing Node (LTS)
+# https://github.com/nodesource/distributions/blob/master/README.md#debinstall
+USER gitpod
+RUN curl -fsSL https://deb.nodesource.com/setup_lts.x | sudo -E bash -
```

Comment:
It'll be nicer to install `Node` via [nvm](https://github.com/nvm-sh/nvm), so people can switch between node versions. 
Looking at the `workspace/full` image, gipod also uses `nvm`. 
https://github.com/gitpod-io/workspace-images/blob/f2d623ca9d270c2ce8560d2ca0f9ce71b105aff2/full/Dockerfile#L120-L134

Reply:
done!